packages/components/src/search-control/README.md

```diff
@@ -8,6 +8,8 @@ SearchControl components let users display a search control.
 1. [Development guidelines](#development-guidelines)
 2. [Related components](#related-components)
 
+   Check out also the [Storybook page](https://wordpress.github.io/gutenberg/?path=/docs/components-spinner--docs) for a visual exploration of this component.
```

Comment:
Nit: This would read better as "Also check out the" or just "Check out the".

Reply:
Thanks for looking over this, @apeatling . I removed the word 'also'